filament/src/View.cpp

```diff
@@ -275,6 +275,14 @@ bool View::isScreenSpaceRefractionEnabled() const noexcept {
     return upcast(this)->isScreenSpaceRefractionEnabled();
 }
 
+void View::setStencilBufferEnabled(bool enabled) noexcept {
+    upcast(this)->setStencilBufferEnabled(enabled);
+}
+
+bool View::getStencilBufferEnabled() const noexcept {
+    return upcast(this)->getStencilBufferEnabled();
+}
```

Comment:
don't forget the java and js bindings

Reply:
Yes :) I'll add them in a separate PR once we have Material APIs to actually do something interesting with the stencil buffer.